### ISSUE TITLE
[FEATURE]: Add title elements to pages

### ISSUE THREAD

**Issue by irisvw** [fdnd-agency/nebulaxplorer#262]
## Description
Each page should have a descriptive title element. This is important for screen readers and SEO.

## Documentation
https://svelte.dev/docs/svelte/svelte-head
